package.json

```diff
@@ -42,7 +42,7 @@
   "lint-staged": {
     "*.{js,jsx,ts,tsx}": [
       "eslint --ext .ts --fix",
-      "tsc --esModuleInterop --noemit",
+      "tsc --esModuleInterop",
```

Comment:
should this be included in the downgrade? is this interconnected?

Reply:
No harm in having it here, it's a minor change. tsconfig.json already has noemit. I can also just remove this change, doesn't matter either way.